server/services/maps/custom.ts

```diff
@@ -59,7 +59,27 @@ export default class CustomMapService {
             }
           }
 
-          locations.push(star);
+          locations.push({
```

Comment:
Merge `master` into your branch will remove this from the changeset. No big deal if it's too painful.